src/management/index.js

```diff
@@ -2015,27 +2015,33 @@ utils.wrapPropertyMethod(ManagementClient, 'getJob', 'jobs.get');
 
 /**
  * Given a path to a file and a connection id, create a new job that imports the
- * users contained in the file and associate them with the given connection.
+ * users contained in the file or JSON string and associate them with the given
+ * connection.
  *
  * @method    importUsers
  * @memberOf  module:management.ManagementClient.prototype
  *
  * @example
  * var params = {
  *   connection_id: '{CONNECTION_ID}',
- *   users: '{PATH_TO_USERS_FILE}'
+ *   users: '{PATH_TO_USERS_FILE}',
+ *   upsert: true, //optional
+ *   send_completion_email: false //optional
  * };
  *
- * management.get(params, function (err) {
+ * management.importUsers(params, function (err) {
  *   if (err) {
  *     // Handle error.
  *   }
  * });
  *
- * @param   {Object}    data                Users import data.
- * @param   {String}    data.connectionId   Connection for the users insertion.
- * @param   {String}    data.users          Path to the users data file.
- * @param   {Function}  [cb]                Callback function.
+ * @param   {Object}    data                        Users import data.
+ * @param   {String}    data.connectionId           Connection for the users insertion.
+ * @param   {String}    data.users                  Path to the users data file.
+ * @param   {String}    data.users_json             JSON data for the users.
+ * @param   {String}    data.upsert                 OPTIONAL: set to true to upsert users, defaults to false
```

Comment:
the way you tell a property is optional in JSDocs is using `[]` (see `cb` below for an example). Should look like this


- `[data.users_json]` -> Whether to update users if they already exist (true) or to ignore them (false).
- `[data.upsert]` -> Whether to send a completion email to all tenant owners when the job is finished (true) or not (false).

See that I changed the description of those params and also removed the default values, as those are not present in the [official docs](https://auth0.com/docs/api/management/v2#!/Jobs/post_users_imports) at least.

Reply:
Fixed by https://github.com/auth0/node-auth0/pull/469/commits/f0b7a2c791145e3b6a02db125bd0d7ef369f46de and https://github.com/auth0/node-auth0/pull/469/commits/52d4024ec60caf0b097dcad69534dc3fadbeb425.

Though I removed the default values as you say, I believe they should be described in the document. This is because the default values are determined by [the library's implementation](https://github.com/auth0/node-auth0/blob/v2.23.0/src/management/JobsManager.js#L167-L168), not by the official API.